src/modules/project/context/ProjectProvider.tsx

```diff
@@ -207,7 +206,7 @@ export const ProjectProvider = ({ projectId, children }: { children: React.React
       {project && isProjectOwner && (
         <>
           <ProjectCreatorModal {...creatorModal} />
-          <MilestoneAdditionModal {...milestonesModal} onSubmit={onMilestonesSubmit} project={project} />
+          {/* <MilestoneAdditionModal {...milestonesModal} onSubmit={onMilestonesSubmit} project={project} /> */}
```

Comment:
do we have a new page for goal creation ? or a different modal?

Reply:
We have a new modal for project creation GoalModal ! The Milestones were so embedded in in the project that it was not worth it to refactor it to goals.